tests/unit/test_tiler.py

```diff
@@ -36,11 +36,11 @@ def it_constructs_from_args(self, level, mpp, request):
         _init = initializer_mock(request, RandomTiler)
 
         random_tiler = RandomTiler(
-            (512, 512), 10, level, 7, True, "", ".png", int(1e4), mpp=mpp
+            tile_size=(512, 512), n_tiles=10, level=level, mpp=mpp
```

Comment:
why are we removing parameters?

Reply:
cause they're the default one and not mandatory, so for sake of readability i'd remove them and use kwargs instead of positional